src/main/webapp/WEB-INF/jsp/welcome.jsp

```diff
@@ -93,10 +93,9 @@
     <br/>
     <%--    Rating Related Buttons - Nichita--%>
     <a class="btn btn-default" href='<spring:url value="/ratings/new" htmlEscape="true"/>'>Rate Pet</a>
```

Comment:
What's the difference between View All Ratings and View Pet Ratings?

Reply:
View All Ratings displays ratings for all pets while View Pet Ratings redirects you to the page where you can select a specific pet and view ratings for it.